nexus/src/app/sagas/instance_migrate.rs

```diff
@@ -127,6 +132,7 @@ impl NexusSaga for SagaInstanceMigrate {
 
         builder.append(reserve_resources_action());
         builder.append(allocate_propolis_ip_action());
+        builder.append(create_migration_record_action());
```

Comment:
I would consider doing this step and the next in the other order--insert the destination VMM and then the migration record so that it's always safe to interpret the target ID in the migration as an FK into the VMM table.